<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControllerArea</class>
 <widget class="QWidget" name="ControllerArea">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>130</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="2" column="5">
    <widget class="QLabel" name="labelValue3">
     <property name="text">
      <string notr="true">-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="ComboCC" name="comboControl3">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QDial" name="knob3">
     <property name="maximum">
      <number>127</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="StyledSlider" name="sliderPressure">
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="StyledSlider" name="sliderSensitivity">
     <property name="maximum">
      <number>1500</number>
     </property>
     <property name="pageStep">
      <number>100</number>
     </property>
     <property name="value">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="push4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="iconSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="labelValue1">
     <property name="text">
      <string notr="true">-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="labelValue2">
     <property name="text">
      <string notr="true">-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QDial" name="knob2">
     <property name="maximum">
      <number>127</number>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="ComboCC" name="comboControl2">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="ComboCC" name="comboControl4">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="labelValue4">
     <property name="text">
      <string notr="true">off</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="StyledSlider" name="sliderPitchWheel">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="value">
      <number>64</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QDial" name="knob1">
     <property name="maximum">
      <number>127</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="ComboCC" name="comboControl1">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pitch wheel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelWheelValue">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sensitivity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSensitivityValue">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Pressure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPressureValue">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboCC</class>
   <extends>QComboBox</extends>
   <header>combocc.h</header>
  </customwidget>
  <customwidget>
   <class>StyledSlider</class>
   <extends>QSlider</extends>
   <header>styledslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
